adapter/Cargo.toml

```diff
@@ -14,13 +14,10 @@ serde = { version = "^1.0", features = ['derive'] }
 serde_json = "1.0"
 serde-hex = "0.1.0"
 # Ethereum
-web3 = { git = "https://github.com/samparsky/rust-web3" }
+web3 = { git = "https://github.com/tomusdrw/rust-web3" }
 eth_checksum = "0.1.1"
```

Comment:
Are we sure this works without the version upgraded? I think it's this version that doesn't support tuples

Reply:
@samparsky It seems that the `contract.call()` (https://tomusdrw.github.io/rust-web3/web3/contract/struct.Contract.html#method.call) can take Tuples now.
The `Tokenize` trait is implemented for tuples as well - https://tomusdrw.github.io/rust-web3/web3/contract/tokens/trait.Tokenize.html

Question: These `contract.call()` calls were the reason for the fork, right?
https://github.com/AdExNetwork/adex-validator-stack-rust/blob/82e13d286ffde9e4017ffd5b0341832be0fc7560/adapter/src/ethereum.rs#L626-L638
https://github.com/AdExNetwork/adex-validator-stack-rust/blob/82e13d286ffde9e4017ffd5b0341832be0fc7560/adapter/src/ethereum.rs#L693-L702